### DISCUSSION TITLE
Tray icon timer on macos

### DISCUSSION THREAD

**Discussion by Q-back** [ivan-lednev/obsidian-day-planner#484] [Features & bugs]
Hi. Did you consider (or maybe even try to implement) the tray icon (timer) similar to the one we see inside the Obsidian app?
I wonder I'd be able to contribute and add it to this plugin
![image](https://github.com/ivan-lednev/obsidian-day-planner/assets/25378062/79f0bec6-24c3-4f46-978d-803898cd48e2)

**Comment by ivan-lednev** [comment]
Hey! I don't have a Mac to test this, moreover, this would require some knowledge of a different platform (Swift, MacOS), which is completely outside my ballpark.

**Reply by Q-back** [reply]
Hi. Thanks for feedback. Can't promise but I'll try to spike on that as I really enjoy this extension 🔥
